.circleci/config.yml

```
@@ -628,6 +636,70 @@ jobs:
           command: |
             echo "Nightly build run"
 
+  # -------------------------
+  #    JOBS: Windows
+  # -------------------------
+  windows_job:
+    executor: 
+      name: win/default
+    environment:
+      - ANDROID_HOME: "C:\\android-sdk-windows"
+      - ANDROID_NDK: "C:\\android-sdk-windows\\android-ndk-r19c"
+      - ANDROID_BUILD_VERSION: 28
+      - ANDROID_TOOLS_VERSION: 29.0.2
+      - GRADLE_OPTS: -Dorg.gradle.daemon=false
+      - SDK_TOOLS_URL: https://dl.google.com/android/repository/sdk-tools-windows-3859397.zip
+      - NDK_TOOLS_URL: https://dl.google.com/android/repository/android-ndk-r19c-windows-x86_64.zip
+    steps:
+      - restore_cache_checkout:
+          checkout_type: windows 
+      - run:
+          name: Install Android SDK Tools
+          command: choco install android-sdk
+      - run:
+          name: Setup SDK
+          command: sdkmanager --licenses > nul
```

```suggestion
          command: sdkmanager --licenses
```

Current CI failed here, maybe just remove `> nul`